src/sources/heroku_logs.rs

```diff
@@ -44,12 +44,15 @@ use crate::{
 #[derive(Clone, Debug)]
 pub struct LogplexConfig {
     /// The address to listen for connections on.
```

Comment:
```suggestion
    /// The socket address to listen for connections on.
```

Thoughts? 
That is what I have in my vector source PR :shrug:

Reply:
Unsure? We should probably just make a executive decision, for example - [datadog_agent](https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources/base/datadog_agent.cue#L24) does the same `The address to listen...`.

I'm wondering if it was originally caused by `vector` being TCP socket based, and these two examples being HTTP. Though, that seems less likely checking the [socket](https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources/base/socket.cue#L5) which also doesn't mention "socket".

The underlying type is a "SocketAddress," though I'm not sure if one or the other is more generally known.

Reply:
Updated in [c3f780d](https://github.com/vectordotdev/vector/pull/15680/commits/c3f780d468b03d8d2c5b1ba267791b9121811c22)